.github/workflows/tests.yml

```diff
@@ -14,7 +14,10 @@ jobs:
       fail-fast: true
       matrix:
         php: [7.2, 7.3, 7.4]
-        laravel: [^7.0]
+        laravel: [^7.0, ^8.0]
+        exclude:
+          -   php: 7.2
+              laravel: 8.0
```

Comment:
Think you have some spaces too much here

Reply:
Hm this does work on another repo...

Reply:
Caret sign was missing it seems.

Reply:
Yeah right ;-) Thanks!